internal/service/agreements/init.go

```diff
@@ -12,10 +12,14 @@ import (
 //go:embed approved_researcher.md
 var approvedResearcherMarkdown string
 
+//go:embed study_owner.md
+var studyOwnerMarkdown string
+
```

Comment:
Where is the actual text so that it can be added here?

Reply:
public version: https://isms.arc.ucl.ac.uk/agreement_templates/rism15-study_agreement/
markdown https://github.com/UCL-ARC/research-data-isms/blob/main/docs/Controlled_Documents/Agreement_Templates/RISM15-Study_Agreement.md